src/apikey/lib/middlewares/auth.js

```diff
@@ -1,10 +1,14 @@
 const { apikey } = require('config');
 
 /**
- * @param {Object} req - HTTP request
- * @param {Object} res - HTTP response
- * @param {function} next - do the following
- * @returns {Object|function} res or next
+ * Authentication middleware that checks if the content of the x-api-key header
+ * matches the environment variable used as password.
+ *
+ * @param {Object} req - HTTP request.
+ * @param {Object} res - HTTP response.
+ * @param {function} next - Do the following in error handler.
```

Comment:
The type is `Function` not `function`
```suggestion
 * @param {Function} next - Do the following in error handler.
```